src/code/Utils.cs

```diff
@@ -732,6 +783,45 @@ public static string GetInstalledPackageName(string pkgPath)
             }
         }
 
+        /// <summary>
+        /// Checks if any of the package versions are already installed and if they are removes them from the list of packages to install.
+        /// </summary>
+        internal static HashSet<string> GetInstalledPackages(List<string> pathsToSearch, PSCmdlet cmdletPassedIn)
+        {
+            // Package install paths.
+            // _pathsToInstallPkg will only contain the paths specified within the -Scope param (if applicable).
+            // _pathsToSearch will contain all resource package subdirectories within _pathsToInstallPkg path locations.
+            // e.g.:
+            // ./InstallPackagePath1/PackageA
+            // ./InstallPackagePath1/PackageB
+            // ./InstallPackagePath2/PackageC
+            // ./InstallPackagePath3/PackageD
+
+            // Get currently installed packages.
+            var getHelper = new GetHelper(cmdletPassedIn);
+            var pkgsInstalledOnMachine = new HashSet<string>(StringComparer.CurrentCultureIgnoreCase);
+
+            foreach (PSResourceInfo installedPkg in getHelper.GetPackagesFromPath(
+                name: new string[] { "*" },
+                versionRange: VersionRange.All,
+                pathsToSearch: pathsToSearch,
+                selectPrereleaseOnly: false))
+            {
+                string pkgNameVersion = CreateHashSetKey(installedPkg.Name, installedPkg.Version.ToString());
```

Comment:
```suggestion
                string pkgNameVersion = string.Format("{0}{1}", installedPkg.Name, installedPkg.Version.ToString());
```

Dont need a function for it I think.